CenterForOpenScience/ember-osf-web
# [EMB-508][Collections] Fix unnecessary bold fonts on submit/edit view.

## Purpose

This PR fixes unnecessary bold fonts for "Project Metadata" and "Collection Metadata" sections on the collection submit/edit view.

## Summary of Changes

Moved some sections out of the `<label>` tag. Fix implicit this on template.

## QA Notes

On the submit/edit view, bold fonts should now only apply to the field labels, bot the fields themselves.

## Ticket

https://openscience.atlassian.net/browse/EMB-508

# Reviewer Checklist
